translate/packages/translate/translate/index.js

```diff
@@ -65,8 +80,10 @@ async function _command(params, commandText, secrets = {}) {
 
   try {
     if (targetLang.length != 2) {
-      const ISO6391 = require('iso-639-1');
       targetLang = ISO6391.getCode(targetLang);
+      if (!targetLang) {
+        return {response_type: 'in_channel', text: languageHelp};
```

Comment:
```suggestion
        return {response_type: 'ephemeral', text: languageHelp};
```

Since this is error/help text, ephemeral makes more sense.